packages/web-components/fast-foundation/src/horizontal-scroll/README.md

```diff
@@ -0,0 +1,123 @@
+---
+id: horizontal-scroll
+title: fast-horizontal-scroll
+sidebar_label: horizontal-scroll
+custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/horizontal-scroll/README.md
+---
+
+An implementation of a content scroller as a web-component.
+
+## Setup
+
+### Basic Setup
+
+```ts
+import {
+    provideFASTDesignSystem,
+    fastHorizontalScroll
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+    .register(
+        fastHorizontalScroll()
```

Comment:
Does this need `fastFlipper()` registered as well?

Reply:
Yes. Good catch. Update incoming.